mercury.spec

```diff
@@ -15,6 +15,7 @@ URL:      http://mercury-hpc.github.io/documentation/
 Source0:  https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
 Patch0:   https://github.com/daos-stack/mercury/cpu_usage.patch
 Patch1:   https://github.com/daos-stack/mercury/cxi_provider_plus_daos_9561.patch
+Patch2:   https://github.com/daos-stack/mercury/na_ucx_changes.patch
```

Comment:
does the RPM spec do some magic here because these URLs don't work in a browser

Reply:
They won't until this lands.  But because these files exist locally in the PR, they are findable and used in the build.  Local files always take precedence over URLs.  URLs are still useful here though so that they can be used to fetch the files if they are not already local, say, to somebody who grabs the `mercury.spec`.  Rather than manually grabbing each of the Source and Patch files, they can use something like `spectool -g mercury.spec` to have them all fetched automatically.